terraform-aws-github-runner/modules/runners/policies/lambda-scale-cycle.json

```diff
@@ -0,0 +1,44 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+      {
+        "Effect": "Allow",
+        "Action": [
+          "ec2:DescribeInstances",
+          "ec2:DescribeTags",
+          "ec2:RunInstances",
+          "ec2:CreateNetworkInterface",
+          "ec2:DescribeNetworkInterfaces",
+          "ec2:DeleteNetworkInterface",
```

Comment:
Are these actually needed?  I do see that scale up has these as well, but it seems odd that they'd require this permission.

Reply:
I think this is techincally what is needed to create an EC2 instance

Reply:
I believe this is only necessary if we need more complex networking outside of what EC2 instance launch creates when you create a new EC2 instance. Like an instance with multiple NICs connected to more than 1 network.